site/content/en/topics/shared-ip.md

```diff
@@ -47,3 +47,17 @@ spec:
CtZgD3uyqnpUL6sxWNn755vp5fqe01LS/zwiyM3eHDgL19cFJNueSN322OHHjHt4vGRrUiRt+rgOO24IrO89Ihh+bFUcZ2Wt2itYd6JFwIECBAgQIAAAQIECBAgQIAAAQIEalqg3QfHSXfGvFVxTBaCpkraSksVvN/6s+Fx/qeGNgS/lWMHf296pLWFU9vUQ/VS1fFf3PpGfvz5C8bFPlllcapG7v2N5/OgNVXvzr167436Pf2OmfHD/50X6bvrbtwvv765l0pwnK6ZfdWE2D7ru9LWZ1XDQy56IVJldWqNq4rTvs/884y8MjmFyqnSObVUudzr6/X3miqYX/3mH0Ph/IQPXtKDBdMDBlOrLIeRtquD4/137h1PnTs27d6gtcZwgw58IECAAAECBAgQIECAAAECBAgQIECgZgVqIjiu6KbANy2fMHtxXWVXHJg9tO7hM8ZsEMj2ygLgVDmblmF46PQxDedWNuYsqYvdvlVfyVtZrmLqH1bGXtkyFamlauO0dETj9vOn50cKj1NbdO0++bIOjc9p/LkSHG8uoE3rC6elIjYXRk9+4J34h0fnbnB82pyVsedV9fd6x1+NiuM+Mqjx1+afF69cGwPOm5JvX/f5ETH50B3y7erg+N9OHJWvv9y4g9YYNu7DZwIECBAgQIAAAQIECBAgQIAAAQIEalOgpoLjRJzW9r3t9+/HmffMysPhtG/iTr2yqtlxeYXwguVrY7sL6sPSdKy59p3P7RjnZVXLdz+7MI69rb4Kublr0vFnzx8X+43o1eypleD42Gzd4jv/etRG5x9325tx17MLYvTg7nl1dOMTrs9C43Oz8Di19TdNzN+rK6Zfzh5ulx66t7nW79zn8zWdqx+wVx0cP51VG38kqzqubq01rO7DNgECBAgQIECAAAECBAgQIECAAAECtStQc8FxhTot7/DRG16N17KHzKX242zN4bTWb3pY3IS/r6/GrZzb1HtlLeMbH5sbX7uvPqBt6vzKsScnj40Dsmrn5lolOD7hgEGRqnsbty/++M2445kFkdYhfiVbcqJxq17LuBIcV9/r/Gv2iUGbWes59bXzpS/m6zkfnT0c8JfZWs+pVQfH73x7Quw4oFu+v/LSWsPK9d4JECBAgAABAgQIECBAgAABAgQIEKhtgZoNjhP7zAWrY+RlL+Uz8OUDt4uffnmXWLJqXfTPqmxTuz5bnmFTS07kBz94SUtEpIfGVVfxPpdVE48ftvkq3nRpty4fPGmuurNNbG+N4PiBKYuyNZ9fz78t3eu+TVQ+dzvnuXxN5HMP2yGu/fMR+TXVwfHsbN3kYdn6ydWttYbVfdgmQIAAAQIECBAgQIAAAQIECBAgQKB2BWo6OE7slbV4D9ujX/z6zN3ymajsO37/QXH7SaPyfc29VFfZVqqXm7umJce3RnCcqqxTv6l9d9KIOOeQ+rWLG99P9Xk/ySqyT8wqslNrLjhO57TGMF2nESBAgAABAgQIECBAgAABAgQIECBQ+wLtOjh+6KVF2cPv+sTQfl03KV39QLvLjxwelx05LD/v4O9Nj8dnLM23p18yPnYb0mOT189P6yF/sMzDqjXro+/k5/Pq3OFZBe5bV+y1yari9NC9tM5y7+6dN9ln451bIzhem93A4AtfiEUr1kbPbp3jvav3jr49Nr6fSbe8EfdPWZjf0qxsSYoRHyxJ0ZLguDWGjcfuMwECBAgQIECAAAECBAgQIECAAAECtSnQroPj9OC4tITEqR8bHKl6+E9G9Ynu2RIRS7PlKFKofObdsyKtdZza/31jbBy0S/2aw8/MXB77X/dKvn9wn65xz8mj45Dd++afU+j7xOtL4/Q7ZsbBY/rGzX+5c74/vVz76zlx/oPv5p8PzNYvvju7bpftuuefU7Cc1iL+u3tnxY+OHxmT9h2Y72/uZWsEx+k773luYXzhR/UP89tpYLd46PQxDUtWLFu9Ls7KHh6YHiKY2qVHDIsrjhqeb6eXlgTHrTVs+BIbBAgQIECAAAECBAgQIECAAAECBAjUrEC7D47venZBs7jXZWsZTz50w+UaUnD6T799r+HaVJm7fZ8u+YPiKjtP+9j2GwTHdWvXx8Rrp+UP2KucM6BXl+jRtVPMXVIfUKf9954yepsHx+k+TvzZW/GzJ+enzbylMfbv2XmDe/34rn3i0bN2zwP3ynktCY7Tua0xrHyHdwIECBAgQIAAAQIECBAgQIAAAQIEalegXQfHj01fGpMfeCeeziqIN9VSVfAFnx662RD3zqxC+NRfvJ1XKDe+PoXGl2SVuKlat7qlZSAuemh23PDo3HzZiupjaTmIiw8fFmd9ckj0aeFSFWOvnBqvzl0VJ2XrC6e1kxu3L/30zbj9qQWx1/Ce8eKFezY+HP+Yhd9nZyF4autvmrjR8VSRfVpWPV2pvK6ckO71ms/tGF/9xJDKrob3VK392ZvrH643N1vmYkjfTS8Fki5ojWHDF9kgQIAAAQIECBAgQIAAAQIECBAgQKAmBdp1cFwRTctEvDJ3ZUO1cAp7Rw7qHgOzauCWtFQtPOXdFdG5U8Su2/fIw+Ku6UMTbX22pMXr76+Kl+eszNZB7hqjB3ePYf26RaemL2uix617aEG2XvOLs1dEdtsxfljPrLp682Fwa+6kNYat+R7XECBAgAABAgQIECBAgAABAgQIECCw7QVqIjje9kzugAABAgQIECBAgAABAgQIECBAgAABAh1HQHDccebaSAkQIECAAAECBAgQIECAAAECBAgQINAiAcFxi5icRIAAAQIECBAgQIAAAQIECBAgQIAAgY4jIDjuOHNtpAQIECBAgAABAgQIECBAgAABAgQIEGiRgOC4RUxOIkCAAAECBAgQIECAAAECBAgQIECAQMcREBx3nLk2UgIECBAgQIAAAQIECBAgQIAAAQIECLRIQHDcIiYnESBAgAABAgQIECBAgAABAgQIECBAoOMI/D80r5oDBjfHPgAAAABJRU5ErkJggg==)
```

Comment:
seem the image is not visible. Move it an image file and refer with file name here?